dask_ml/_typing.py

```diff
@@ -16,6 +16,6 @@
 FrameOrSeriesUnion = Union[DataFrame, Series, dd.Series, dd.DataFrame]
 SeriesType = Union[dd.Series, Series]
 DataFrameType = Union[DataFrame, dd.DataFrame]
-Number = Union[int, float, np.float64, np.int64, np.int32]
+Number = Union[int, np.float64, np.float64, np.int64, np.int32]
```

Comment:
```suggestion
Number = Union[int, np.float64, np.int64, np.int32]
```

np.float64 is already in this list.

Reply:
closes #850